### DISCUSSION TITLE
api request speed is very slow, is there any solution？

### DISCUSSION THREAD

**Discussion by li-yu-tfs** [perplexityai/api-discussion#51] [Q&A] [answered]
I found that the website was also slow the first time, but after each time answered questions quickly ， But I need to call this API, https://api.perplexity.ai/chat/completions, each time is slow, how to solve?

**Comment by rafs2006** [comment]
Hey @li-yu-tfs! Does it respond slowly if you are on another network, too? And could you please elaborate a bit more on 'very slow'.

**Comment by aarashy** [comment, marked as answer]
I'd also recommend using sonar-small if latency is your priority
